src/Compilers/CSharp/Portable/CodeGen/EmitExpression.cs

```diff
@@ -2286,7 +2286,7 @@ internal static bool MayUseCallForStructMethod(MethodSymbol method)
             var containingType = method.ContainingType;
             // overrides in structs that are special types can be called directly.
```

Comment:
>// overrides in structs that are special types can be called directly. [](http://example.com/codeflow?start=12&length=70)

Consider aligning the comment with implementation, i.e. saying that a subset of special types can be assumed to keep the method.